### PR TITLE
Support multi arch

### PR DESCRIPTION
**Pull request by Mmduh-483**
Add ppc64le  daemonset and add ppc64le CI job

### CODE REVIEW

**Comment by lennybe** [comment]
recheck

**Comment by Mmduh-483** [comment]
@zshi-redhat @ahalim-intel can you guys please review this patch?

**Comment by Mmduh-483** [comment]
@adrianchiris

**Comment by lennybe** [comment]
recheck

**Comment by Mmduh-483** [comment]
@zshi-redhat @ahalim-intel Do have comments for this patch?

File: .travis.yml [hunk 1]
```diff
@@ -20,16 +35,26 @@ deploy:
...
       branch: master
   # Push image to Dockerhub on tag
   - provider: script
     skip_cleanup: true
     script: >
       bash -c '
-      docker tag nfvpe/sriov-cni nfvpe/sriov-cni:$TRAVIS_TAG;
-      docker push nfvpe/sriov-cni:$TRAVIS_TAG;
+      if [ "${ARCH}" == "amd64" ]; then
+        docker push nfvpe/sriov-cni:$TRAVIS_TAG
```
Comment (zshi-redhat) [inline .travis.yml]: Does the tag build have TRAVIS_TAG appended to sriov-cni image automatically? or do we need to tag the image to TRAVIS_TAG?

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+
 go:
   - "1.10"
+
 env:
   global:
   - PATH=$GOROOT/bin:$GOPATH/bin:$PATH
 
+before_install:
+  - sudo systemctl restart docker
```
Comment (zshi-redhat) [inline .travis.yml]: Why this is required?

File: Dockerfile.ppc64le [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+RUN apk add --update --virtual build-dependencies build-base linux-headers make && \
+    cd /usr/src/sriov-cni && \
+    make clean && \
+    make build
+
+FROM ppc64le/alpine
+COPY --from=builder /usr/src/sriov-cni/build/sriov /usr/bin/
+WORKDIR /
+
+LABEL io.k8s.display-name="SR-IOV CNI"
```
Comment (zshi-redhat) [inline Dockerfile.ppc64le]: nit: add ppc64le in label?

File: .travis.yml (hunk 2 above)
Comment (Mmduh-483) [reply .travis.yml]: The CI for ppc64le failed to fetch filed when run the command to install the required packages, and restarting the docker service is the solution for it.
Capture

File: Dockerfile.ppc64le (hunk 3 above)
Comment (Mmduh-483) [reply Dockerfile.ppc64le]: fixed

File: .travis.yml (hunk 1 above)
Comment (Mmduh-483) [reply .travis.yml]: It should be add by the script, seems I removed the line docker tag nfvpe/sriov-cni nfvpe/sriov-cni:$TRAVIS_TAG; by mistake

**Comment by Mmduh-483** [comment]
@zshi-redhat Are there other comments?

**Comment by zshi-redhat** [comment]
> @zshi-redhat Are there other comments?

@Mmduh-483  it looks good to me.
@ahalim-intel would you please take a look?

**Comment by martinkennelly** [comment]
@Mmduh-483 Are you still working on this PR? Do you want it reviewed?

**Comment by Mmduh-483** [comment]
> @Mmduh-483 Are you still working on this PR? Do you want it reviewed?

No, it is not needed anymore, I will close it

**Pull request closed without merging**